content/docs/guides/multi-target-exporter.md

```diff
@@ -351,24 +353,24 @@ docker \
 
 This command works similarly to [running the blackbox exporter using a config file](#run-exporter).
 
-If everything worked, you should be able to go to [localhost:9090/targets](http://localhost:9090/targets) and see under `blackbox-exporter-metrics` an endpoint with the state `UP` in green. If you get a red `DOWN` make sure that the blackbox exporter you started [above](run-exporter) is still running. If you see nothing or a yellow `UNKNOWN` you are really fast and need to give it a few more seconds before reloading your browser’s tab.
+If everything worked, you should be able to go to [localhost:9090/targets](http://localhost:9090/targets) and see under `blackbox` an endpoint with the state `UP` in green. If you get a red `DOWN` make sure that the blackbox exporter you started [above](run-exporter) is still running. If you see nothing or a yellow `UNKNOWN` you are really fast and need to give it a few more seconds before reloading your browser’s tab.
 
-To tell Prometheus to query `"localhost:9115/probe?target=prometheus.io&module=http_2xx"` you add another scrape job where you set the `metrics_path` to `/probe` and the parameters under `params:` in the Prometheus config file `prometheus.yml`:
+To tell Prometheus to query `"localhost:9115/probe?target=prometheus.io&module=http_2xx"` you add another scrape job `blackbox-http` where you set the `metrics_path` to `/probe` and the parameters under `params:` in the Prometheus config file `prometheus.yml`:
 
 <a name="prometheus-config"></a>
 
 ```yaml
-scrape_configs:
-
-- job_name: blackbox-exporter-metrics
+global:
   scrape_interval: 5s
+
+scrape_configs:
+- job_name: blackbox # to get metrics about the exporter itself
```

Comment:
Be consistent in capitalisation of your comments, they've varying across the file.